### PR TITLE
Added remediation description, fixed policy title and validate rego (details in the description.)

### PR DESCRIPTION
**Pull request by rezoan**
- dbauditingsettings.rego
- dbfirewallrules.rego
- keyvaultsecrets.rego 
and its related master-compliance-test.json

### CODE REVIEW

File: azure/terraform/dbauditingsettings.rego [hunk 1]
```diff
@@ -34,11 +34,11 @@ mssql_log_retention = false {
     azure_attribute_absence["mssql_log_retention"]
 }
 
-mssql_log_retention_err = "Azure SQL Server audit log retention is less than 91 days" {
+mssql_log_retention_err = "Azure SQL Server audit log retention is not grater then 90 days" {
```
Comment (farchide) [inline azure/terraform/dbauditingsettings.rego:37]: @rezoan `greater`

File: azure/terraform/dbauditingsettings.rego [hunk 2]
```diff
@@ -47,9 +47,9 @@ mssql_log_retention_metadata := {
     "Type": "IaC",
     "Product": "AZR",
     "Language": "Terraform",
-    "Policy Title": "Azure SQL Server audit log retention is less than 91 days",
+    "Policy Title": "Azure SQL Server audit log retention should be grater then 90 days",
```
Comment (farchide) [inline azure/terraform/dbauditingsettings.rego:50]: `greater`

File: azure/terraform/dbfirewallrules.rego [hunk 3]
```diff
@@ -46,11 +46,11 @@ db_firewall = false {
     azure_attribute_absence["db_firewall"]
 }
 
-db_firewall_err = "SQL Server Firewall rules allow access to any Azure internal resources" {
+db_firewall_err = "SQL Server Firewall rules configuration currently allowing everyone get full access" {
```
Comment (farchide) [inline azure/terraform/dbfirewallrules.rego:49]: `SQL Server Firewall rule configuration allowing full inbound access to everyone`

**Review by farchide — CHANGES_REQUESTED** [review]

File: azure/terraform/dbauditingsettings.rego (hunk 1 above)
Comment (rezoan) [reply azure/terraform/dbauditingsettings.rego]: fixed.

File: azure/terraform/dbauditingsettings.rego (hunk 2 above)
Comment (rezoan) [reply azure/terraform/dbauditingsettings.rego]: fixed.

File: azure/terraform/dbfirewallrules.rego (hunk 3 above)
Comment (rezoan) [reply azure/terraform/dbfirewallrules.rego]: updated.

**Review by rezoan — COMMENTED** [review]
changes has been made as per suggestion.

**Review by farchide — APPROVED** [review]

**Pull request merged**